tutorial/01_Introduction/intro.ipynb

```diff
@@ -8,7 +8,7 @@
    "source": [
```

Comment:
<p>mention if you can modify this in place or it is read-only. If it is read-only, how do I modify it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hpc4cmb/toast/pull/369/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>Good point, some options are fixed by the OS runtime environment of python, but some can be changed after toast is imported.  Will give more details.</p>

Reply:
<p>Added text about setting log level manually or through environment.  Same with threading.</p>